check.py

```diff
@@ -360,7 +360,7 @@ def run_binaryen_js_tests():
     if not s.endswith('.js'): continue
     print s
     f = open('a.js', 'w')
-    binaryen_js = open(os.path.join(options.binaryen_bin, 'binaryen.js')).read()
+    binaryen_js = open(os.path.join(options.binaryen_root, 'bin', 'binaryen.js')).read()
```

Comment:
This is unrelated too, but something i needed to get out-of-tree testing working.

Reply:
Seems harmless to leave in then?